rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1761,7 +1761,7 @@ s64 SPUThread::get_ch_value(u32 ch)
 
 		if (mask1 & SPU_EVENT_LR && raddr)
 		{
-			if (mask1 != SPU_EVENT_LR)
+			if (mask1 != SPU_EVENT_LR && mask1 != SPU_EVENT_LR | SPU_EVENT_TM)
```

Comment:
imo better to just drop the throw, the only situation where it can throw is when both events `TM`, `LR` and `SN` (or only `LR` and `SN`)  are enabled in mask, which is ok because events `TM` and `LR can be signaled and break the stall, so its not a deadlock.

Reply:
temporary solution  break out lines! Core i7 3770k
 		if (mask1 & SPU_EVENT_LR && raddr)
 		{
                }
Dante's Inferno and Metal Gear Solid Collection Back normal!